engine/install/ubuntu.md

```diff
@@ -44,11 +44,11 @@ Docker Engine is supported on `x86_64` (or `amd64`), `armhf`, `arm64`, and `s390
 
 ### Uninstall old versions
 
-Older versions of Docker were called `docker`, `docker.io`, or `docker-engine`.
+Older versions of Docker were called `docker`, `docker.io`, `docker-engine` or `docker-compose-pligin`.
```

Comment:
@akankshakumari393 There's a typo: `plugin`. Also, I don't think any older version of Docker was called 'docker-compose-plugin`, therefore, I don't think it makes sense to add Docker Compose in this line. 

Perhaps we can add a new line recommending users to uninstall Docker Compose if it already exists. I believe in this case, they should uninstall `docker-compose` and not `docker-compose-plugin` ? @glours could you please confirm?